compiler/damlc/tests/src/DA/Test/IncrementalBuilds.hs

```diff
@@ -148,7 +149,7 @@ tests damlc repl = testGroup "Incremental builds"
               createDirectoryIfMissing True (takeDirectory $ dir </> file)
               writeFileUTF8 (dir </> file) content
           let dar = dir </> "out.dar"
-          callProcessSilent damlc ["build", "--project-root", dir, "-o", dar, "--incremental=yes"]
```

Comment:
Should those stay non-silent or is this a debug left-over?

Reply:
good catch! fixed